lib/package.json

```diff
@@ -1,11 +1,14 @@
 {
-  "name": "victory-native",
-  "version": "40.0.4",
+  "name": "@sunrun/victory-native-xl",
+  "version": "0.0.4",
```

Comment:
We haven't discussed versioning, but I think resetting our version to zero will make it difficult to trace which version of the upstream lib we're on. 

I'm leaning towards matching our package version to the latest version of the upstream lib that we have merged. So `40.0.4`.

Reply:
we can change. 
But every time we run publish, it has to use a new version. So if we make changes, and update package with new version, we will diverge from 40.0.4 very soon.  That is why I start with 0.0.1

Reply:
What about `40.0.4-0.0.1` ? I think it's in the spirit of [semver](https://semver.org/)

Reply:
Per github package [versioning](https://github.com/NuGet/docs.microsoft.com-nuget/blob/main/docs/concepts/Package-Versioning.md), I can change name to 40.0.4-1